WiseBuy/View Controllers/BarcodeScanViewController.m

```diff
@@ -27,24 +27,23 @@ @interface BarcodeScanViewController () <AVCaptureVideoDataOutputSampleBufferDel
 
 @implementation BarcodeScanViewController
 
--(void)viewDidLoad{
+- (void)viewDidLoad{
     [super viewDidLoad];
 
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(checkPermissions)
                                                  name:UIApplicationDidBecomeActiveNotification
                                                object:nil];
 }
--(void)viewDidAppear:(BOOL)animated{
+- (void)viewDidAppear:(BOOL)animated{
     [super viewDidAppear:animated];
 
     self.alreadyScanned = NO;
     
     [self checkPermissions];
 }
 
-- (void)dealloc
-{
+- (void)dealloc {
```

Comment:
Love the change to have consistent styling for brackets - depending on the codebase, some engineerings but function brackets on the same line as the method names and others put it on a new line. 

We typically put it on a new line but either is fine as long as you are consistent